spec/mail/mailbox.mdown

```diff
@@ -34,6 +34,27 @@ A **Mailbox** object has the following properties:
   0 <= sortOrder < 2^31.
 
     A mailbox with a lower order should be displayed before a mailbox with a higher order (that has the same parent) in any mailbox listing in the client's UI. Mailboxes with equal order SHOULD be sorted in alphabetical order by name. The sorting SHOULD take into account locale-specific character order convention.
+- **sharedWith**: `Rights[String]`
+  Used to share a mailbox with other users. This is a map of username to `Rights`, where `Rights` is an array of single char strings among these values (adapted from [@!RFC4314]):
+    - "l": mailbox is visible but not its content
+    - "r": read the messages in this mailbox
+    - "w": modify keywords of the messages in this mailbox
+    - "i": append or copy a message to this mailbox
+    - "k": create child mailbox
+    - "x": delete this mailbox
+    - "t": move a message from this mailbox to Trash folder
+    - "e": destroy a message
+    - "a": modify this sharedWith property
+
+  Example:
+
+      sharedWith: {
+        "alice@example.com": ["e", "i", "l", "r", "t", "w"],
+        "bob@example.com": ["l", "r"]
```

Comment:
Stupid question can you "r" without "l" if yes what does that mean?

Reply:
it theory, you could be able to read this message (by its messageId), but not the mailbox containing it